src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,11 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed === true ? 'TodoInfo--completed' : ''}`}>
```

Comment:
Use logical operators instead. BTW you can use a classnames library here 
```suggestion
  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
```